content/applications/finance/fiscal_localizations/italy.rst

```diff
@@ -441,19 +626,88 @@ issued to the public administrations must contain:
+selling cleaning services for a public building) the :abbr:`PA (Public Administration)` business
+will send the VAT to the Tax Agency themselves, and the vendor just has to use an appropriate
+:guilabel.
+
+In the Italian Localization we have configured a special :guilabel:`Fiscal Position` to deal with
+partners belonging to the :abbr:`PA (Public Administration)`.
+
+.. figure:: italy/italy-split-payment-fp.png
+   :align: center
+   :alt: Public Administration Fiscal Position
+
+   Public Administration Fiscal Position
+
+Process
+-------
+
+.. _italy/digital-signature:
+
+1. Digital qualified signature
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+For invoices and bills intended to the **Pubblica Amministrazione (B2G)**, a **Digital Qualified
+Signature** is required for all files sent through the :abbr:`SdI (Sistema di Interscambio)`. The
+**XML** file must be certified using a certificate that is either:
+
+- a **smart card**;
+- a **USB token**;
+- an :abbr:`HSM (Hardware Security Module)`.
+
+.. warning::
+   Odoo **cannot** digitally sign documents for you. When we detect a :guilabel:`Codice
+   Destinatario` that is 6 digits long, then the :abbr:`EDI (Electronic Data Interchange)` process
+   stops, the invoice is set on the :guilabel:`Requires user signature` state. You can download the
+   document in **XML**, sign it with any :guilabel:`Digital Qualified Signature` provider's external
+   program and send it through the :abbr:`AdE (Agenzia Delle Entrate)` portal.
+
+    .. figure:: italy/italy-edi-requires-user-signature.png
+       :align: center
+       :alt: EDI Request User Signature State
+
+       EDI Request User Signature State
+
+2. Acceptance or Refusal
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. warning::
+   As Odoo does not handle sending signed invoices to :abbr:`PA (Public Administration)` businesses,
+   you won't be able to trigger those states from within Odoo, but you have to deal with the
+   :abbr:`AdE (Agenzia Delle Entrate)` portal. Yet, if you will still be able to receive
+   notifications about it, which will advance the invoice :guilabel:`SdI State` to the correct one.
+
+After receiving the invoice through the :abbr:`SdI (Sistema di Interscambio)`, the :abbr:`PA (Public
+Administration)` business has **15 days** to **accept** the invoice. If it does, then the process
+ends here. If the :abbr:`PA (Public Administration)` business **refuses** the invoice, it is still
+considered valid once it was accepted by the :abbr:`SdI (Sistema di Interscambio)` so you'll have to
+emit a credit note to compensate.
+
+2b. Expired Terms
+~~~~~~~~~~~~~~~~~
+
+If the :abbr:`PA (Public Administration)` business doesn't reply in the **15 days** time frame,
+please contact the :abbr:`PA (Public Administration)` business directly, sending them both the
+invoice and the received deadline notification by email. You will be able to get an arrangement with
+them, and manually set the correct :guilabel:`SdI State` on your invoice.
```

Comment:
If the :abbr:`PA (Public Administration)` business doesn't reply within 15 days,
you need to contact the :abbr:`PA (Public Administration)` business directly, sending them both the
invoice and the received deadline notification by email. You can get an arrangement with
them, and manually set the correct :guilabel:`SdI State` on your invoice.